source/options.html

```diff
@@ -2,20 +2,52 @@
 <meta charset="UTF-8" />
 <title>GhostText options</title>
 <link rel="stylesheet" href="npm:webext-base-css">
+<link rel="stylesheet" href="options.css">
 <form id="options-form">
-	<label>
+	<label for="serverPort">
 		Server Port
+	</label>
+	<input
+		name="serverPort"
+		id="serverPort"
+		type="number"
+		min="1"
+		max="65536"
+		step="1"
+		placeholder="4001"
+		value="4001"
+		required
+	/>
+	<br>
+	<div class="field">
 		<input
-			name="serverPort"
-			type="number"
-			min="1"
-			max="65536"
-			step="1"
-			placeholder="4001"
-			value="4001"
-			required
+			name="notifyOnConnect"
+			id="notifyOnConnect"
+			type="checkbox"
+			value="notify"
+			checked=true
 		/>
-	</label>
+		<label for="notifyOnConnect">
+			Display notification popup on connect/disconnect with the text editor
+		</label>
+	</div>
+	<br>
+	<div class="field">
+		<input
+			name="focusOnDisconnect"
+			id="focusOnDisconnect"
+			type="checkbox"
+			value="focus"
+			checked=true
+		/>
+		<label for="focusOnDisconnect">
+			Focus browser's tab when text editor is disconnected
+			<br/>
```

Comment:
This simulates 
 - point 1
 - point 2

instead of simulating (point 1 point 2)

it also makes the option name slightly easier to read without the extra info

So it's not about universal label width, the previous one is auto-wrapped because it doesn't have this semantic separation

Reply:
There's no need, punctuation already does that job

Reply:
It does, but with slightly less clarity

<img width="386" alt="gt44" src="https://user-images.githubusercontent.com/12956286/222406673-0f48ec59-3900-4312-996c-075d7520fb29.png">

Anyway, removed the breaks